polyfill/test/zoneddatetime.mjs

```diff
@@ -2811,10 +2811,10 @@ describe('ZonedDateTime', () => {
       );
     });
     it('compares time zone IDs if exact times are equal', () => {
```

Comment:
I would suggest changing the descriptions of these tests (e.g. "disregards time zone IDs if exact times are equal")